### ISSUE TITLE
Beta 2.0 (62): Can't scroll new integrations

### ISSUE THREAD

**Issue by wixoff** [home-assistant/home-assistant-iOS#410]
In build 62 of the beta app, the list of new integrations (brought up by pressing the orange "+" on the integrations page of HA) doesn't scroll.  I can drag my finger up and down but nothing visible happens until I turn my phone sideways then back again.  Then the list might or might not be at a new position.

This worked fine in earlier builds.

I'm using iOS 13 public beta 1, if that might be an issue.  On an iPhone 7 Plus.

**Comment by SeanPM5** [comment]
I reproduced this issue on iOS 13 dev b2. It also happens when in mobile Safari so this is almost certainly a problem introduced on Apple's end rather than this build of the app. There's a bunch of bugs in mobile Safari on iOS 13 beta at the moment.

I'll tag this with a new iOS 13 label so we can keep track of issues like this.

**Comment by wixoff** [comment]
Yes, it scrolls properly for me too on iOS 13.1b.  I concur with closing this.
